cmd/ndm_daemonset/probe/blkidprobe.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package probe
+
+import (
+	"github.com/openebs/node-disk-manager/blockdevice"
+	"github.com/openebs/node-disk-manager/cmd/ndm_daemonset/controller"
+	"github.com/openebs/node-disk-manager/pkg/blkid"
+	"k8s.io/klog"
+)
+
+const (
+	blkidProbePriority = 2
```

Comment:
If we decide to have a new probe, I would suggest a probe with lower priority i.e higher number, since lower numbers are reserved for udev and sysfs probes